modules/apps/site-initializer/site-initializer-cms/src/main/resources/site-initializer/fragments/group/cms/fragments/sidebar/index.css

```diff
@@ -16,6 +16,7 @@
 		border-right: 1px solid #e7e7ed;
 		display: flex;
 		grid-area: sidebar;
+		margin-top: var(--control-menu-height);
```

Comment:
If we wrap `lfr-drop-zone` with `<div class="position-sticky">` and set a `z-index` and `top` properties we can remove these `margin-top`'s and keep it strictly on the nav. I don't know if we can pass a class into `lfr-drop-zone`. I tried `class` but not `className`.

Reply:
Thanks @pat270!
I believe this  `lfr-drop-zone` is only available when we are editing the page. If we're not editing, this styles will not be applied. I am not entirely sure about this